docs/src/common_links.inc

```diff
@@ -8,8 +8,8 @@
 .. _cirrus-ci: https://cirrus-ci.com/github/SciTools/iris
 .. _codespell: https://github.com/codespell-project/codespell
 .. _conda: https://docs.conda.io/en/latest/
-.. _contributor: https://github.com/SciTools/scitools.org.uk/blob/master/contributors.json
-.. _core developers: https://github.com/SciTools/scitools.org.uk/blob/master/contributors.json
+.. _contributor: https://github.com/SciTools/iris/graphs/contributors
+.. _core developers: https://github.com/orgs/SciTools/teams/iris-devs/members
```

Comment:
This is only visible to people within the GitHub SciTools organization - at ANY level. That covers a lot of people who might be reading the docs, but I'm not sure. I don't have a better alternative, maybe this is an argument for a docs page like others have!

Reply:
I think a docs for this would make sense. I can see how it might be useful.